src/pages/home/_components/AppsGrid.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import { useColorMode } from '@docusaurus/theme-common';
+import clsx from 'clsx';
+import styles from './AppsGrid.module.scss';
+
+export default function AppsGrid({ list }) {
+  const { isDarkTheme } = useColorMode();
+  return (
+    <div>
+      <div className={styles.appsContainer}>
+        {list.map((item) => (
+          <a
+            href={item.href}
+            key={item.image}
+            className={clsx(styles.appCard, 'card')}
+          >
+            <div className="avatar card__body">
+              <img
+                className={clsx(
+                  'avatar__photo',
+                  styles.appLogo,
+                  isDarkTheme && item.isMonochrome && styles.darkModeInvert
+                )}
+                src={item.image}
+                alt={item.name}
```

Comment:
It's best to keep this alt here but empty (`alt=""`). Because the image is part of a link, the alt text of the image becomes part of the link, leading to duplication (the link is then read out as eg "Polypane Polypane".) So in this case, keeping it empty helps understanding.